### PR TITLE
Add Ukrainian index page

### PR DESCRIPTION
**Pull request by kazanzhy**
I translated the English index page into Ukrainian

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -87,6 +87,9 @@ collections:
   th: # Thai translations
     output: true
     permalink: /:collection/:path.html
+  uk: # Ukrainian translations
```
Comment (Roman-Smachylo-Imprivata) [inline _config.yml:90]: Why 'uk' and not 'ua'? 
Usually 'uk' uses for United Kingdom.

File: _uk/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        icon: "fa fa-cloud"
+        link: /online-courses.html
+      - title: Книги
+        description: "Друковані та цифрові книги по Scala."
+        icon: "fa fa-book"
+        link: /books.html
+      - title: Посібники
+        description: "Путівник з серії кроків для створення програм на Scala."
+        icon: "fa fa-tasks"
+        link: /tutorials.html
```
Comment (Roman-Smachylo-Imprivata) [inline _uk/index.md:89]: Why previous titles were written in quotes and these ones without?

File: _uk/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      - title: "Посібники та огляди"
+        description: "Поглиблена документація, що покриває багато особливостей Scala."
+        icon: "fa fa-database"
+        link: /overviews/index.html
+      - title: "Довідник по стилю"
+        description: "Поглиблений посібник з написання ідіоматичного коду на Scala."
+        icon: "fa fa-bookmark"
+        link: /style/index.html
+      - title: "Шпаргалка"
+        description: "Зручна шпаргалка, з основ синтаксису Scala."
```
Comment (Roman-Smachylo-Imprivata) [inline _uk/index.md:106]: We don't need comma here.

File: _config.yml (hunk 1 above)
Comment (kazanzhy) [reply _config.yml:90]: It's a language code and might be a little bit confusing.
Also, there are country codes and there are pairs like `en-us`, en-uk`, `en-ca`, fr-ca` and `uk-ua`
https://www.loc.gov/standards/iso639-2/php/code_list.php

File: _uk/index.md (hunk 3 above)
Comment (kazanzhy) [reply _uk/index.md:106]: Done

File: _uk/index.md (hunk 2 above)
Comment (kazanzhy) [reply _uk/index.md:89]: Missed them. Fixed

**Review by anatoliykmetyuk — APPROVED** [review]

**Review by julienrf — APPROVED** [review]
Thank you for your help! The structure looks good to me. However, please note that we are in the process of removing the “Scala 2” / “Scala 3” split on the index page. We will soon update the main index page, but we are not going to change the translations, so please keep an eye on what is going on and backport the changes on the Ukrainian index page!

**Comment by kazanzhy** [comment]
@julienrf thanks.
I'll keep the Ukrainian version close to the English one.
I'll be very glad if you tag me under the new index page PR

**Pull request merged**